Xamarin.Essentials/Contacts/ContactActivity.android.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Android.App;
+using Android.OS;
+using Android.Provider;
+using Android.Runtime;
+using Content = Android.Content;
+
+namespace Xamarin.Essentials
+{
+    [Activity(ConfigurationChanges = Content.PM.ConfigChanges.Orientation | Content.PM.ConfigChanges.ScreenSize)]
+#pragma warning disable IDE0040 // Add accessibility modifiers
+    internal class ContactActivity : Activity
```

Comment:
Please check out `IntermediateActivity` and see if you can use that instead of creating another activity.